### PR TITLE
fix health monitor regression after #557

### PR DESCRIPTION
**Pull request by churkin**
\cc @miherlosev @kirovboris

### CODE REVIEW

**Comment by kirovboris** [comment]
lgtm

File: src/client/sandbox/xhr.js [hunk 1]
```diff
@@ -21,6 +23,10 @@ export default class XhrSandbox extends SandboxBase {
         this.corsSupported = typeof xhr.withCredentials !== 'undefined';
     }
 
+    static isOpenedXhr (obj) {
+        return obj[IS_OPENED_XHR];
+    }
+
```
Comment (miherlosev) [inline src/client/sandbox/xhr.js]: We can detect that xhr request is opened using `readyState` property.
After calling `xhr.open(...)` the `readyState` become equal 1.
So you can rewrite method isOpenedXhr as:
```javascript
static isOpenedXhr (obj) {
       return isXhr(obj) && obj.readyState >= 1 ;
 }
```

**Comment by miherlosev** [comment]
\r-

File: src/client/sandbox/xhr.js (hunk 1 above)
Comment (churkin) [inline src/client/sandbox/xhr.js]: What is `isXhr` function?

**Comment by miherlosev** [comment]
After discussing with @churkin:
There is a problem with `isXhr` checking.
So a new flag `hammerhead|xhr|is-opened-xhr` is a good solution for this.

lgtm.

**Pull request merged**